llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -173,17 +173,17 @@ cl::opt<module_split::IRSplitMode> SplitMode(
 cl::opt<bool> DoSymGen{"symbols", cl::desc("generate exported symbol files"),
                        cl::cat(PostLinkCat)};
 
-enum SpecConstMode { SC_USE_RT_VAL, SC_USE_DEFAULT_VAL };
+enum SpecConstLowerMode { SC_USE_NATIVE, SC_USE_EMULATION };
 
-cl::opt<SpecConstMode> SpecConstLower{
+cl::opt<SpecConstLowerMode> SpecConstLower{
     "spec-const",
     cl::desc("lower and generate specialization constants information"),
     cl::Optional,
-    cl::init(SC_USE_RT_VAL),
-    cl::values(
-        clEnumValN(SC_USE_RT_VAL, "rt", "spec constants are set at runtime"),
-        clEnumValN(SC_USE_DEFAULT_VAL, "default",
-                   "set spec constants to C++ defaults")),
+    cl::init(SC_USE_NATIVE),
+    cl::values(clEnumValN(SC_USE_NATIVE, "native",
+                          "lower spec constants to native spirv instructions"),
+               clEnumValN(SC_USE_EMULATION, "emulation",
+                          "replace spec constants with emulation technique")),
```

Comment:
i would personally like if we had also left info about rt/c++ default spec constants in description, because now it's mostly describing the implementation, not the meaning. does it make sense?